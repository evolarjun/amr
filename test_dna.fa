>contig1 blaTEM-156_cds
AACCCCTATTTGTTTATTTTTCTAAATACATTCAAATATGTATCCGCTCATGATACAATAACCCTGATAA
ATGCTTCAATAATATTGAAAAAGGAAGAGTATGAGTATTCAACATTTCCGTGTCGCCCTTATTCCCTTTT
TTGCGGCATTTTGCCTTCCTGTTTTTGCTCACCCAGAAACGCTGGTGAAAGTAAAAGATGCTGAAGATCA
GTTGGGTGCACGAGTGGGTTACATCGAACTGGATCTCAACAGCGGTAAGATCCTTGAGAGTTTTCGCCCC
GAAGAACGTTTTCCAATGATGAGCACTTTTAAAGTTCTGCTATGTGGCGCGGTATTATCCCGTGTTGACG
CCGGGCAAGAGCAACTCGGTCGCCGCATACACTATTCTCAGAATGACTTGGTTGAGTACTCACCAGTCAC
AGAAAAGCATCTTACGGATGGCATGACAGTAAGAGAATTATGCAGTGCTGCCATAACCATGAGTGATAAC
ACTGCGGCCAACTTACTTCTGACAACGATCGGAGGACCGAAGGAGCTAACCGCTTTTTTGCACAACATAG
GGGATCATGTAACTCGCCTTGATCGTTGGGAACCGGAGCTGAATGAAGCCATACCAAACGACGAGCGTGA
CACCACGATGCCTGCAGCAATGGCAACAACGTTGCGCAAACTATTAACTGGCGAACTACTTACTCTAGCT
TCCCGGCAACAATTAATAGACTGGATGGAGGCGGATAAAGTTGCAGGACCACTTCTGCGCTCGGCCCTTC
CGGCTGGCTGGTTTATTGCTGATAAATCTGGAGCCGGTGAGCGTGGGTCTCGCGGTATCATTGCAGCACT
GGGGCCAGATGGTAAGCCCTCCCGTATCGTAGTTATCTACACGACGGGGAGTCAGGCAACTATGGATGAA
CGAAATAGACAGATCGCTGAGATAGGTGCCTCACTGATTAAGCATTGGTAACTGTCAGACCAAGTTTACT
CATA

>contig2 blaPDC-114_blast_cds
ATGCGCGATACCAGATTCCCCTGCCTGTGCGGCATCGCCGCTTCCACACTGCTGTTCGCCACCACCCCGG
CCATTGCCGGCGAGGCCCCGGCGGATCGCCTGAAGGCACTGGTCGACGCCGCCGTACAACCGGTGATGAA
GGCCAATGACATTCCGGGCCTGGCCGTAGCCATCAGCCTGAAAGGAGAACCGCATTACTTCAGCTATGGG
CTGGCCTCGAAAGAGGACGGCCGCCGGGTGACGCCGGAGACCCTGTTCGAGATCGGCTCGGTGAGCAAGA
CCTTCACCGCCACCCTCGCCGGCTATGCCCTGACCCAGGACAAGATGCGCCTCGACGACCGCGCCAGCCA
GCACTGGCCGGCACTGCAGGGCAGCCGCTTCGACGGCATCAGCCTGCTCGACCTCGCGACCTATACCGCC
GGCGGCTTGCCGCTGCAGaTCCCCGACTCGGTGCAGAAGGACCAGGCACAGATCCGCGACTACTACCGCC
AGTGGCAGCCGACCTACGCGCCGGGCAGCCAGCGCCTCTATTCCAACCCGAGCATCGGCCTGTTCGGCTA
TCTCGCCGCGCGCAGCCTGGGCCAGCCGTTCGAACGGCTCATGGAGCAGCAAGTGTTCCCGGCACTGGGC
CTCGAACAGACCCACCTCGACGTGCCCGAGGCGGCGTTGGCGCAGTACGCCCAGGGCTACGGCAAGGACG
ACCGCCCGCTACGGGCCGGTCCCGGCCCGCTGGATGCCGAAGGCTACGGGGTGAAGACCAGCGCGGCCGA
CCTGCTGCGCTTCGTCGATGCCAACCTGCATCCGGAGCGCCTGGACAGGCCCTGGGCGCAGGCGCTCGAT
GCCACCCATCGCGGTTACTACAAGGTCGGCGACATGACCCAGGGCCTGGGCTGGGAAGCCTACGACTGGC
CGATCTCCCTGAAGCGCCTGCAGGCCGGCAACTCGACGCCGATGGCGCTGCAACCGCACAGGATCGCCAG
GCTGCCCGCGCCACAGGCGCTGGAGGGCCAGCGCCTGCTGAACAAGACCGGTTCCACCAACGGCTTCGGC
GCCTACGTGGCGTTCGTCCCGGGCCGCGACCTGGGACTGGTGATCCTGGCCAACCGCAACTATCCCAATG
CCGAGCGGGTGAAGATCGCCTACGCCATCCTCAGCGGCCTGGAGCAGCAGGGCAAGGTGCCGCTGAAGCG
CTGA

>contig3 blaOXA-436_partial_cds
GCCACAATTGGTCAGCAGTTTGCTATTGAAAGCCCGCCAAGACGGTGCTACTCTTAGCGCCTCATTTTTATGGATTTATTGCATTAGGCAAGGGGACATTATGCGTGCGTTAGCCTTATCGGCTGTGTTGATGGTGACAACGATGATTGGCATGCCTGCGGTGGCAAAGGAGTGGCAAGAGAACAAGAGTTGGAATGCTCACTTTAGCGAACATAAAACCCAAGGCGTGGTTGTGCTCTGGAACGAGAATACACAGCAGGGTTTTACCAACGATCTTAAACGGGCAAACCAAGCATTTTTACCTGCATCGACCTTTAAGATCCCAAACAGTTTAATTGCCTTGGACTTAGGTGTGGTTAAGGATGAGCATCAAGTCTTTAAATGGGATGGACAGACGCGAGATATCGCCGCGTGGAATCGCGACCATGACTTAATCACCGCGATGAAGTATTCGGTTGTGCCTGTTTATCAAGAATTTGCCCGCCAAATTGGCGAGGCCCGTATGAGTAAAATGTTGCACGCCTTCGATTATGGTAATGAGGATATCTCGGGCAATTTGGACAGTTTTTGGCTCGATGGTGGTATTCGCATTTCGGCTACCCAGCAAATCGCTTTTTTACGCAAGCTGTACCACAACAAGTTGCACGTTTCTGAGCGTAGTCAGCGCATCGTTAAACAAGCCATGCTGACCGAGGCAAATGCCGACTATATCATCCGGGCGAAAACTGGCTATTCGGTCAGAATTGAACCGAAAATCGGTTGGTGGGTTGGCTGGATCGAACTGGATGACAATGTGTGGTTCAAGTGGTTAGCGGCGCATTTGTGTAAAATAGCCGTCATATAAGCTGTAAAGTTATATGGACAAAATACTTATAGTCGATGCGCTTATCGCTAATGGCTCA


>contig4 vanG_cds exact
ATGACTGGTAGTCAGACGGAAAAAGAATTGTATGTCAACCAATGTAAAATAGCCTATAAGCTACCCGATG
GTGTAAAAATTGAAGAAAGAGGTGTGTAAAATGCAAAATAAAAAAATAGCAGTTATTTTTGGAGGCAATT
CAACAGAGTACGAGGTGTCATTGCAATCGGCATCCGCTGTTTTTGAAAATATCAATACCAATAAATTTGA
CATAATTCCAATAGGAATTACAAGAAGTGGTGAATGGTATCACTATACGGGAGAAAAGGAGAAAATCCTA
AACAATACTTGGTTTGAAGATAGCAAAAATCTATGCCCTGTTGTCGTTTCCCAAAATCGTTCCGTTAAAG
GCTTTTTAGAAATTGCTTCAGACAAATACCGTATTATAAAAGTTGATTTGGTATTCCCCGTATTGCATGG
CAAAAACGGCGAAAATGGGACTTTGCAGGGCATATTTGAATTGGCAGGAATACCTGTTGTTGGCTGCGAT
ACACTCTCATCAGCTCTTTGTATGGATAAGGACAGGGCACATAAACTCGTTAGCCTTGCGGGTATATCTG
TTCCTAAATCGGTAACATTCAAACGCTTTAACGAAGAAGCAGCGATGAAAGAGATTGAAGCGAATTTAAC
TTATCCGCTGTTTATTAAACCTGTTCGTGCAGGCTCTTCCTTTGGAATAACAAAAGTAATTGAAAAGCAA
GAGCTTGATGCTGCCATAGAGTTGGCATTTGAACACGATACAGAAGTCATCGTTGAAGAAACAATAAACG
GCTTTGAAGTCGGTTGTGCCGTACTTGGCATAGATGAGCTCATTGTTGGCAGAGTTGATGAAATCGAACT
GTCAAGCGGCTTTTTTGATTATACAGAGAAATATACGCTTAAATCTTCAAAGATATATATGCCTGCAAGG
ATTGATGCCGAAGCAGAAAAACGGATACAAGAAGCGGCTGTAACCATATATAAAGCTCTGGGCTGTTCGG
GTTTTTCCAGAGTGGATATGTTTTATACACCGTCTGGCGAAATTGTATTTAATGAGGTAAACACAATACC
AGGCTTTACCTCGCACAGTCGCTATCCAAATATGATGAAAGGCATTGGTCTATCGTTCTCCCAAATGTTG
GATAAGCTGATAGGTCTGTATGTGGAATGATGAAAACGATTGAGCTTGAAAAGGAAGAAATTTATTGTGG
AAATTTGCTGCTCGTCAACAAAAATTATCCGCTACGAGATAACAATGTAAAGGGTTTAGT

>contig6 gyrA:T86I
TCAAAACTTTGAACTAATAGAAGGATTTTTATGGAGAATATTTTTAGCAAAGATTCTGAT
ATTGAACTTGTAGATATAGAAAATTCTATAAAAGGTAGTTACTTAGACTATTCTATGAGT
GTTATTATAGGTCGTGCTTTGCCTGACGCAAGAGATGGTTTAAAGCCTGTTCATAGAAGA
ATTTTATATGCTATGCAAAATGATGAGGCAAAAAGTAGAACAGATTTTGTCAAATCAGCC
CGTATAGTGGGTGCTGTTATAGGTCGTTATCATCCACATGGAGATATAGCAGTTTATGAT
GCTTTGGTTAGAATGGCTCAAGATTTTTCTATGAGATATCCAAGTATTACAGGACAAGGC
AACTTTGGATCTATAGATGGTGATAGCGCTGCTGCGATGCGTTATACTGAAGCAAAAATG
AGTAAACTTTCTCATGAGCTTTTAAAAGATATAGATAAAGATACGGTCGATTTTGTTCCA
AATTATGATGGTTCAGAAAGTGAACCTGATGTCTTACCTTCTAGGGTTCCAAATTTATTA
TTAAATGGTTCAAGTGGTATAGCTGTAGGTATGGCGACAAACATCCCACCTCATAGTTTA
AATGAGTTGATAGATGGACTTTTATATTTGCTTGATAGTAAAGATGCAAGCTTAGAAGAG
ATTATGCAGTTTATCAAAGGTCCAGATTTTCCAACAGGTGGAATAATTTATGGTAAAAAA
GGTATTATAGAAGCTTATCGCACAGGGCGTGGTCGCGTGAAAGTGCGCGCTAAAACTCAT
ATTGAAAAAAAGACAAATAAAGATGTTATTGTTATAGATGAGCTTCCTTATCAAACCAAT
AAAGCTAGGCTTATAGAACAGATTGCAGAGCTTGTTAAAGAAAAGCAAATTGAAGGAATA
TCTGAAGTAAGAGATGAGAGCAATAAAGAAGGAATCCGCGTTGTTATAGAGCTTAAACGT
GAGGCTATGAGTGAAATTGTTTTAAACAATCTATTTAAATCTACTACTATGGAAAGTACT
TTTGGTGTGATTATGTTGGCAATTCACAATAAAGAACCTAAAATTTTCTCTTTGTTAGAA
CTTTTAAATCTTTTCTTAACCCATAGAAAAACAGTTATTATTAGAAGAACGATTTTTGAA
CTTCAAAAGGCAAGAGCAAGAGCTCATATTTTAGAAGGTCTTAAAATTGCACTTGATAAT
ATAGATGAAGTGATTGCTTTAATTAAAAATAGTTCTGATAATAATACCGCAAGAGATTCT
TTGGTAGCTAAATTTGGTCTTAGTGAGCTTCAAGCCAATGCTATTTTAGATATGAAACTT
GGTCGTTTAACAGGACTTGAAAGAGAAAAAATCGAAAATGAACTTGCAGAATTAATGAAA
GAAATTGCAAGACTTGAAGAAATTTTAAAAAGTGAAACCTTGCTTGAAAATTTAATTCGC
GATGAATTAAAAGAAATTAGAAGTAAATTTGATGTGCCACGTATTACTCAAATTGAAGAT
GATTATGATGATATTGATATTGAAGATTTGATTCCTAATGAAAATATGGTTGTAACTATT
ACGCATCGTGGTTATATTAAGCGTGTGCCTAGTAAACAATATGAAAAGCAAAAACGAGGC
GGAAAAGGAAAATTAGCCGTTACGACTTATGATGATGATTTTATAGAAAGTTTCTTTACG
GCAAATACACATGATACACTTATGTTTGTAACAGATCGTGGACAGCTTTATTGGCTTAAA
GTTTATAAAATTCCTGAAGGCTCAAGAACGGCTAAAGGAAAAGCAGTGGTAAATCTTATT
AATTTACAAGCTGAAGAAAAAATCATGGCTATTATTCCAACCACGGATTTTGATGAGAGT
AAATCTTTATGTTTCTTTACTAAAAATGGTATTGTAAAGCGTACAAATTTGAGTGAATAT
CAAAATATCAGAAGTGTAGGAGTTAGAGCGATCAACTTGGATGAAAATGATGAGTTGGTA
ACTGCTATTATTGTTCAAAGAGATGAAGATGAAATTTTTGCCACTGGTGGTGAAGAAAAT
TTAGAAAATCAAGAAATTGAAAATTTAGATGATGAAAATCTTGAAAATGAAGAAAGTGTA
AGCACACAAGGTAAAATGCTCTTTGCAGTAACCAAAAAAGGTATGTGTATCAAATTCCCA
CTTGCTAAAGTGCGTGAAATCGGCCGTGTAAGTCGTGGGGTGACGGCTATTAAGTTTAAA
GAGAAAAATGACGAATTAGTAGGTGCAGTTGTTATAGAAAATGATGAGCAAGAAATTTTA
AGCATAAGTGCAAAAGGTATAGGAAAACGCACAAATGCTGGAGAATATAGATTGCAAAGC
AGAGGTGGTAAGGGTGTAATTTGTATGAAACTTACAGAAAAAACCAAAGATCTTATTAGC
GTAGTTATAGTAGATGAAACTATGGATTTGATGGCTCTTACAAGTTCAGGCAAGATGATA
CGTGTTGATATGCAAAGCATTAGAAAAGCAGGGCGTAATACGAGTGGTGTCATTGTAGTT
AATGTGGAAAATGACGAGGTGGTTAGCATCGCTAAGTGTCCTAAAGAGGAAAATGATGAG
GATGAGTTAAGCGATGAAAACTTTGGTTTAGATTTGTAATAAGTTTAAA
AGGGCATAAAGGCTCTGTGCAAAAGAAAATTGGTAAGTAAGGGATAAGATATGAGTAAAG
CATTAATTAAATTCATAAGATTATCTTCAACTAAAGCAAGATTGATTGCTAGAGAAGTTC
AGGGGATGAATGCAGAATTAGCTATGGCTAGTTTAAAATTTATGCCAAATAAAGGTGCAA
AATATATCGCAAATGCAATTTCAAGTGCAGTAGCAAATGGTGGCTTTGAAGCAAACGAAG
TGATTGTAAAAAGTTGTCGTGTTGACGCAGCTGCGGTATTAAAAAGATTTAGACCAAGAG
CTAGAGGAAGTGCAAGTCGTATTAGAAAACCAACTTCACATATTTTAGTAGAAGTGGTAA
AAGCGGAAGTGAAAGCTGAAGAGAAAAAAACAGTAGCTAAAAAAACTACTACAACAAAAG
CTCCAGCTAAAAAAACAACAAGCACTAAAAAAGCAACAGCGAAAAAGGAAAGCTAATGGG
ACAAAAAGTTAATCCAATTGGTCTAAGACTAGGAATCAATAGAAAC
>Contig5 Campylobacter 23S:A2075G 
TAAGAGCAAGTTCTAATACAAACTCTACTTGAAAGATTTCTATTAAACTCTAAATATTAT
CTATATTCAAAGTCTTCTTAAAAACCTTAACAAGGAAGTGATGCTTTATTAAAGATAAGC
CAAACGCTCTATTAGTACTGGTCAGCTAAAGGACTTTCATCCATTACACACCCAGCCTAT
CAAACTAGTAGTCTTCTAGAGAGCTTAGAGAAGATTCATCTTAGAGTTGGCTTCACGCTT
AGATGCTTTCAGCGTTTATCCTTTCCAAACTTAGCTACGCTGCGATGCTCTTGGCAGAAC
AACAGCTACACCAGTGGTTTGTTCAACCCGGTCCTCTCGTACTAGGGTCAAATCTCTTCA
ATCTTCTTACGCCCACGGCAGATAGGGACCGAACTGTCTCACGACGTTCTGAACCCAGCT
CGCGTACCGCTTTAAATGGCGAACAGCCATACCCTTGGGACCTGCTCCAGCCCCAGGATG
CGATGAGCCGACATCGAGGTGCCAAACCTCCCCGTCGATGTGAGCTCTTGGGGGAGATCA
GCCTGTTATCCCCGGGGTACCTTTTATCCTTTGAGCGATGGCCCTTCCACACAGAACCAC
CGGATCACTAAGACCGACTTTCGTCTCTGCTTGACTTGTATGTCTTGCAGTTAAGCTGGC
TTATACCTTTATACTCTACGAACGATTTCCAACCGTTCTGAGCCAACCTTTGTAAGCCTC
CGTTATTATTTGGGAGGCGACCGCCCCAGTCAAACTACCCACCAGACATTGTCCCACTTG
AGGATAACTCAAGCTGGTTAGCTACCCAAATAAGAAAGAGTGGTATCTCAACAATGGCTC
ATATACAACTGGCGTCATATACTCAAAGCCTCCCACCTATCCTGCACATTCTTATCCAAA
TAGCAGTGTCAAGCTGTAGTAAAGGTCCACGGGGTCTCTCCGTCTTGCCGCGGGTAGGAG
GAATTTTCACCTCCACTACAATTTCACTGGATCCCTCTTTGAGACAGCTCCCATCTCGTT
ACGCCATTCATGCAGGTCGGTATTTAACCGACAAGGAATTTCGCTACCTTAGGACCGTTA
TAGTTACGGCCGCCGTTTACTCGGGCTTCGATCAAGAGCTTCGCTAATGCTAACCCCATC
AATTAACCTTCGAGCACCGGGCAGGCGTCACACCCTATACATCCTCTTACGAGTTAGCAG
AGTGCTGTGTTTTTGGTAAACAGTCGGGAGGGACTCTTTGTTGTAAGTTTCTTCGCTTTC
GGAGTAAATCCTAATACGAAGCGAACCACACCTTATACCGAAGATACGGTGCTATTTTGC
AGAGTTCCTTAAAGAGAGTTCTTCCACGCGCCTTAGAATACTCATCCCACCCACCTGTGT
CGGTTTACGGTACGGGCAACATTAGCTAAACTTAGAAACTTTTCTTGGCTCGATGGCATC
AGGGGTTCTTCTATCCATCCGAAGACTTCAAAAAGCCTTTCAGTTCTCGGAGTATTGTTA
TACGGATTTGCCTGTATAACCTCCTACGACCTTAGACTAGCACTTCCATCCGCTAGCCCC
CTTAGCCCTAAGCGTCCTTCCATCGCACACTAATGTTGGTATTGGAATATTAACCAATTT
GCCATCGTCTACCCCTTTCGGACTCGACTTAGGACCCGACTAACCCTACGATGACGAGCA
TCGCGTAGGAAACCTTGGGTTTACGGCGTTAATGATTCTCACATTAATTATCGCTACTCA
TGCCTGCATGCTCACTTCTATTCGCTCCAGCACTCCTTACCGGTATACCTTCGACGCAAA
TAGAACGCTCTCCTACCACTTGCACACAACTTAAACTACTTAACTTTAGTCTAAGAATAT
TTTTACTGATTTTACTAAGTTTTATTTCTATCTTTACTCTATAAAAATCAGTAAAAATAT
AACTAAACATATAACTTAGTTTCTCATATTCTAAACTTAAATAATCTAAGTTGTGCGCAA
GTCTACAGCTTCGGTACTTACTTTAGCCCCGTTATATTTTC

>contig7 Campylobacter 50S_L22:A103V  NZ_CUUA01000002.1 Campylobacter jejuni strain OXC6585, whole genome shotgun sequence  59884-60509
AATCACATCGGTTATAAGCTTGGTGAATTTGCTCCAACACGCACATTTAAAGGGCATAAA
GGCTCTGTGCAAAAGAAAATTGGTAAGTAAGGGATAAGATATGAGTAAAGCATTAATTAA
ATTCATAAGATTATCTTCAACTAAAGCAAGATTGATTGCTAGAGAAGTTCAGGGGATGAA
TGCAGAATTAGCTATGGCTAGTTTAAAATTTATGCCAAATAAAGGTGCAAAATATATCGC
AAATGCAATTTCAAGTGCAGTAGCAAATGGTGGCTTTGAAGCAAACGAAGTGATTGTAAA
AAGTTGTCGTGTTGACGCAGCTGCGGTATTAAAAAGATTTAGACCAAGAGCTAGAGGAAG
TGCAAGTCGTATTAGAAAACCAACTTCACATATTTTAGTAGAAGTGGTAAAAGCGGAAGT
GAAAGCTGAAGAGAAAAAAACAGTAGCTAAAAAAACTACTACAACAAAAGCTCCAGCTAA
AAAAACAACAAGCACTAAAAAAGCAACAGCGAAAAAGGAAAGCTAATGGGACAAAAAGTT
AATCCAATTGGTCTAAGACTAGGAATCAATAGAAACTGGGAATCAAGATGGTTTCCAACT
AAAGCAAATTTAGTAGAAAATATCGG

>contigX nimIJ_hmm NZ_JHXM01000017.1:91300-91800 Prevotella brevis ATCC 19188 T433DRAFT_scaffold00015.15_C, whole genome shotgun sequence
CTAATTACCACCGCGCTGCCTTACCAACTCAATAGCCTCTTTACCTGTCAGGTGCTCAATGTCCATACGG
ATGGCCAGCATATGTGCAAGGCCGCTTTCCAGTTCTTTCTGCAAGGCGTCGTCATGGTTGGGATTAAAGC
GATTACCCATCATATGCATGATGGTTCGTTTCTCGGCTTCATCCTCTACAATTTCAATTCTGCCAAAGGC
TATCACACTACGAAAGTAGGTGGTGTACTTCTCTGGTCTCACATCGTCCTTTTCTATAACACAAAAAGAG
GCTTTGTTGCATCTGCGGATGGCATCCACTTTATGGCCAGCCATAGCACTGTGGAAGTATAGTTTGCCGT
TGTCATAGACATAGCTGATGGGCACGGCATAGGGATAGTCATTATCGCCCAACAAGGCTAATGTGCCAGA
AGTAGCCTTCTGCAGAACGGCGATGCTGTCTGCGTCAGTCAGTTGCTGACGCTTTCGTCTCATTTCTCGG
AACTCACTCAT

>contig8 Partial DNA only blaTEM-156
AACCCCTATTTGTTTATTTTTCTAAATACATTCAAATATGTATCCGCTCATGATACAATAACCCTGATAA
ATGCTTCAATAATATTGAAAAAGGAAGAGTATGAGTATTCAACATTTCCGTGTCGCCCTTATTCCCTTTT
TTGCGGCATTTTGCCTTCCTGTTTTTGCTCACCCAGAAACGCTGGTGAAAGTAAAAGATGCTGAAGATCA
GTTGGGTGCACGAGTGGGTTACATCGAACTGGATCTCAACAGCGGTAAGATCCTTGAGAGTTTTCGCCCC
GAAGAACGTTTTCCAATGATGAGCACTTTTAAAGTTCTGCTATGTGGCGCGGTATTATCCCGTGTTGACG
CCGGGCAAGAGCAACTCGGTCGCCGCATACACTATTCTCAGAATGACTTGGTTGAGTACTCACCAGTCAC
AGAAAAGCATCTTACGGATGGCATGACAGTAAGAGAATTATGCAGTGCTGCCATAACCATGAGTGATAAC
ACTGCGGCCAACTTACTTCTGACAACGATCGGAGGACCGAAGGAGCTAACCGCTTTTTTGCACAACATAG
GGGATCATGTAACTCGCCTTGATCGTTGGGAACCGGAGCTGAATGAAGCCATACCAAACGACGAGCGTGA
CACCACGATGCCTGCAGCAATGGCAACAACGTTGCGCAAACTATTAACTGGCGAACTACTTACTCTAGCT

>contig9 >case4_case6_sul2_aph3pp-Ib Providencia rettgeri strain Pret_2032, whole genome shotgun sequence  2160922-2162737  150-1527 (reverse comp'd)
CAAATCGGCATAGCGATCTGCTGTTCCGAGCCGCCCAAGGTCGATCAGACCCGTGCATTGAAGAGTTTTA
GGGTCCACCATGAAGTTCGGCATGCAGGGATCACCATGGCAAACAACCATATCGGTGCGCTCTTGGTCGA
GCCGCACCGGTAGCTCTCGTTCGACACGAGCCAAAAGATCGAGCTGCGGCGTACTCTTGTCCTCGTCCGG
TAAGAAGTCGGGATTGACGGCATTGCGGGACACCACATCAACGGCGCGTCCGAACATTCGCGACAGCCTG
CGCTCAAACGGACATTGATCAACCGATAGGCTGTGAACAGCGCCAAGTTGCTGCCCCATTGACGGCCACG
CTTTGAGCAAATCCGCTCCAGACAGATCAGCCGCCGGTACTCCCGGAATTGCCGTTATCACCAAGCATGC
ACCCTCCTGTTCCTCCTGCCAGTTGATGACCTCGGGGCAAGCCACACCTCGACCTTTGAGCCAAATGAGG
CGGTCACGCTCTCCAGCGAGCTCACCGCGGCGGGAAGCAGGTGCGATTTTCGCGAAGGCATGCCCGTCAC
CACGTCGAAAAACAAAATCACCAGATTCTCCGCCTCTGACAGGCAACCAGTCAGAATGCGATTCACCAAA
AAAAATATTAGTTCGATTCAATGGAGGTTCCTTCAGTTTTCTGATGAAGCGCGAATATAGAGAAATATCC
CGAATGTGCAGTTAACGAATTCTTGCGGTTTCTTTTAGCGCCGCCAATACCGCCAGCCCGTCGCGCAAGG
GGCGCGGCTCGTGTGTGCGGATGAAGTCAGCTCCACCTGCGGCGGCGGCAAGCTCTGCAGCGAGTGTCGC
GGCCCCGACATCCCCCGGACCACGGCCTGTGAGCGCGCGCAGAAAGGATTTGCGCGAAACAGACAGAAGC
ACCGGCAAATCGAAGCGCAGCCGCAATTCATCGAACCGCGCCAGCACCGAGAGCGAGGTTTCGGGAGCAG
CCCCCAGAAAAAACCCCATGCCGGGATCAAGGACAAGGCGGTTGCGTTTGATACCGGCACCCGTCAGCGC
CGCGATGCGCGCGTCAAAGAACGCCGCAATGTGATCCATGATGTCGCCAGCGGGTGCCTCGCGCCGATCT
GCCTGCCCGTCTTGCACCGAATGCATAACGACGAGTTTGGCAGATGATTTCGCCAATTGCGGATAGAACG
CAGCGTCTGGAAAACCGCGAATATCATTGAGATAGGCCACACCACGCGACAAGGCATAGGCTTGCGTCGC
GGGTTGATAACTGTCGAGCGAGACGGGAATGCCATCTGCCTTGAGCGCGTCCAGCACCGGCGCGATACGC
TCGATTTCTGTGTCGGACGAAACAGGCGCGGCGTCGGGGTTGCTGGAT

>contig10 blaOXA_internal_stop >NZ_CP008798.1 Klebsiella pneumoniae subsp. pneumoniae KPNIH24 plasmid pKPC-484, complete sequence  44200-45800
TTGCTCTGCTCAAACGCGAAGGCCGGTGCCCGTCTGACGTTGAACACCGACAGATTAAGT
ACCGGAACAACGTGATTGAATGCGATCATGGCAAACTGAAACGGATAATCGGCGCCACGC
TGGGATTTAAATCCATGAAGACGGCTTACGCCACCATCAAAGGTATTGAGGTGATGCGTG
CACTACGCAAAGGCCAGGCCTCAGCATTTTATTATGGTGATCCCCTGGGCGAAATGCGCC
TGGTAAGCAGAGTTTTTGAAATGTAAGGCCTTTGAATAAGACAAAAGGCTGCCTCATCGC
TAACTTTGCAACAGTGCCAGGCAGGCTTATCTTGGACAAGAAGATCGCTTGGCCTCGCGC
GCAGATCAGTTGGAAGAATTTGTTCACTACGTGAAAGGCGAGATCACCAAGGTAGTCGGC
AAATAATGTCTAAAACAAAGTTATGCACCTATTAAGCGCACAGCGGAGCAATGAAGGATA
CCTTGATGAAAAAAATTTTGCTGCTGCATATGTTGGTGTTCGTTTCCGCCACTCTCCCAA
TCAGTTCCGTGGCTTCTGATGAGGTTGAAACGCTTAAATGCACCATCATCGCAGACGCCA
TTACCGGAAATACCTTATATGAGACCGGAGAATGTGCCCGTCGTGTGTCTCCGTGCTCGT
CTTTTAAACTTCCATTGGCAATCATGGGGTTTGATAGTGGAATCTTGCAGTCGCCAAAAT
CACCTACGTGGGAATTGAAGCCGGAATACAACCCGTCTCCGAGAGATCGCACATACAAAC
AAGTCTATCCGGCGCTATGGCAAAGCGACTCTGTTGTCTAGTTCTCGCAGCAATTAACAA
GCCGTCTGGGAGTTGATCGGTTCACGGAATACGTAAAGAAATTTGAGTACGGTAATCAAG
ATGTTTCCGGTGACTCGGGGAAGCATAACGGCTTGACCCAGTCATGGCTGATGTCGTCGC
TCACCATATCTCCCAAGGAGCAAATTCAGTTTCTTCTACGCTTTGTCGCGCATAAGCTGC
CTGTATCCGAAGCGGCTTATGACATGGCGTATGCCACAATCCCGCAGTACCAGGCAGCCG
AAGGATGGGCTGTACATGGAAAAAGCGGCAGCGGCTGGCTTCGGGACAATAACGGCAAGA
TAAATGAAAGTCGTCCGCAGGGCTGGTTCGTGGGCTGGGCTGAAAAAAACGGACGGCAAG
TTGTTTTCGCCCGATTGGAAATAGGAAAGGAAAAGTCCGATATTCCCGGCGGGTCTAAAG
CACGAGAGGATATTCTCGTGGAATTACCCGTGTTGATGGGTAACAAATGATATGTGGCGT
CATCGAGAGCAGATGCATAACCCTGCGCTCGAGCGGACCTCGCGCATAAAGCCGCGCGAG
TCCGCTCACCTTGAACGTTAGATGCACTAAGCACATAATTGCTCACAGCCAAACTATCAG
GTCAAGTCTGCTTTTATTATTTTTAAGCGTGCATAATAAGCCCTACACAAATTGGGAGAT
ATATCATGAAAAGAGTTTGATGCTCAGGGTGTAGCGGTTCGGTTTATTGACGACGGGATC
AGTACCGACGGTGATATGGGGCAAATGGTGGTCACCATCCT

