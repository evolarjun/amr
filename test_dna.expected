Target identifier	Contig id	Start	Stop	Strand	Gene symbol	Protein name	Method	Target length	Reference protein length	% Coverage of reference protein	% Identity to reference protein	Alignment length	Accession of closest protein	Name of closest protein	HMM id	HMM description
contig1	contig1	101	958	+	blaTEM-156	class A beta-lactamase TEM-156	ALLELEX	286	286	100.00	100.00	286	WP_061158039.1	class A beta-lactamase TEM-156	NF000531.2	TEM family class A beta-lactamase
contig10	contig10	486	1307	+	blaOXA	class D beta-lactamase	INTERNAL_STOP	274	274	100.00	99.64	274	WP_000722315.1	oxacillin-hydrolyzing class D beta-lactamase OXA-9	NF000270.1	class D beta-lactamase
contig2	contig2	1	1191	+	blaPDC	PDC family class C beta-lactamase	BLASTX	397	397	100.00	99.75	397	WP_061189306.1	class C beta-lactamase PDC-114	NF000422.2	PDC family class C beta-lactamase
contig3	contig3	101	802	+	blaOXA	OXA-48 family class D beta-lactamase	PARTIALX	234	265	88.30	100.00	234	WP_058842180.1	OXA-48 family carbapenem-hydrolyzing class D beta-lactamase OXA-436	NF000387.2	OXA-48 family class D beta-lactamase
contig4	contig4	101	1147	+	vanG	D-alanine--D-serine ligase VanG	EXACTX	349	349	100.00	100.00	349	WP_063856695.1	D-alanine--D-serine ligase VanG	NF000091.3	D-alanine--D-serine ligase VanG
contig8	contig8	101	700	+	blaTEM	TEM family class A beta-lactamase	PARTIAL_CONTIG_ENDX	200	286	69.93	100.00	200	WP_061158039.1	class A beta-lactamase TEM-156	NF000531.2	TEM family class A beta-lactamase
contig9	contig9	1	675	-	aph(3'')-Ib	aminoglycoside O-phosphotransferase APH(3'')-Ib	PARTIAL_CONTIG_ENDX	225	275	81.82	100.00	225	WP_109545041.1	aminoglycoside O-phosphotransferase APH(3'')-Ib	NF032895.1	aminoglycoside O-phosphotransferase APH(3'')-Ib
contig9	contig9	715	1377	-	sul2	sulfonamide-resistant dihydropteroate synthase Sul2	PARTIAL_CONTIG_ENDX	221	271	81.55	100.00	221	WP_001043265.1	sulfonamide-resistant dihydropteroate synthase Sul2	NF000295.1	sulfonamide-resistant dihydropteroate synthase Sul2
